docs/concepts/resources/filter_types.py

```diff
@@ -184,39 +184,52 @@
 y = pf.dsp.filter.low_shelve(impulse, 1/4*frequency, -10, 2, 'II')
 pf.plot.freq(y * 10**(-40/20), ax=axis, label='Low-shelve')
 
+# notch filter
+y = pf.dsp.filter.notch(impulse * .1**3, 100, 4)
+pf.plot.freq(y, ax=axis, label='Notch')
+
+y = pf.dsp.filter.notch(impulse * .1**4, 1000, 4)
+pf.plot.freq(y, ax=axis, label='Notch')
+
+y = pf.dsp.filter.notch(impulse * .1**5, 10000, 4)
+pf.plot.freq(y, ax=axis, label='Notch')
+
 axis.set_title('')
-axis.set_xlabel('')
 axis.set_xlim(20, 20e3)
-axis.set_ylim(-70, 20)
-axis.legend(loc=4, ncol=3)
+axis.set_ylim(-140, 20)
+axis.legend(loc='lower center', ncol=3, bbox_to_anchor=(.5, -.4))
+
+plt.savefig('filter_types_audio_filter_1.png', dpi=150)
+
+# %% audio filter (bell and shelves) ------------------------------------------
+_, ax = plt.subplots(1, 1, figsize=(15/2.54, 8/2.54), sharey=True)
 
 # shelve cascades
-axis = ax[1]
+axis = ax
 gain = 10
 y, _, ideal = pf.dsp.filter.high_shelve_cascade(
     impulse, 125, 'lower', gain, None, 5)
-pf.plot.freq(ideal, c=[0, 0, 0, .5], ax=axis, ls="--", label="Ideal response")
+# pf.plot.freq(ideal, c=[0, 0, 0, .5], ax=axis, ls="--", label="Ideal response")
```

Comment:
What's the reason to keep the commented code?

Reply:
yes, removed those